qlobber-sub.js

```diff
@@ -23,14 +23,26 @@ QlobberSub.prototype._add_value = function (clientMap, val) {
   topicMap.set(val.topic, val.qos)
 }
 
-QlobberSub.prototype._add_values = function (dest, clientMap) {
+QlobberSub.prototype._add_values = function (dest, clientMap, topic) {
   for (var clientIdAndTopicMap of clientMap) {
-    for (var topicAndQos of clientIdAndTopicMap[1]) {
-      dest.push({
-        clientId: clientIdAndTopicMap[0],
-        topic: topicAndQos[0],
-        qos: topicAndQos[1]
-      })
+    var clientId = clientIdAndTopicMap[0]
+    var topicMap = clientIdAndTopicMap[1]
+    if (topic === undefined) {
+      for (var topicAndQos of topicMap) {
+        dest.push({
+          clientId: clientId,
+          topic: topicAndQos[0],
+          qos: topicAndQos[1]
+        })
+      }
```

Comment:
I also remember that sometime when playing around in some codes, assigning to objects was so slow :(